addons/binding/org.openhab.binding.lametrictime/ESH-INF/thing/countdown-app.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="lametrictime"
+                          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                          xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+                          xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+    <thing-type id="countdownApp" listed="true">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="device" />
+        </supported-bridge-type-refs>
+        
+        <label>LaMetric Time Countdown App</label>
+        <description>Provides control for the countdown app on the LaMetric Time</description>
+        
+        <channels>
+            <channel id="configure" typeId="parameterActionType">
+                <label>Configure</label>
+                <description>Configure the timer</description>
+            </channel>
+            <channel id="pause" typeId="triggerActionType">
```

Comment:
Unfortunately, we can't really combine these. The device does not provide state as to whether it is paused or not.  Sending `start` while it is running does nothing. Sending `pause` while it is paused, does nothing. Without separating these, we can easily become out of phase (due to other input, like the mobile app or the device itself). If that happens, the user has to flip the switch more than once to see any effect. Keeping them separate means the user can choose the exact functionality required.

Reply:
Hm, it is really unfortunate that one cannot get the current states from the device.
Nonetheless, I would claim that combining those two channels makes sense, since at least, people can use a toggle switch, if they only control it remotely (and by toggling twice they will in any case get the intended result).

Anyhow, it does not make sense to have two commands (ON and OFF) for every single trigger here. If we say that the state does not make any sense (as it gets out of sync too easily), we should actually not model any separate channels at all, but rather have a single "command channel", which would take Strings like PLAY/PAUSE/RESET (and the user could have a single button for each of those options in the UI).

Which way would you prefer?

Reply:
I like the idea of a command channel - the toggles always felt clumsy to me. I'd love to see something like a momentary button in the future (like a trigger channel in reverse) as part of the core, but this is probably a good solution for now. 

I'll try this out. Let me know what you think of the result.